modules/must-gather-storage-limit.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="must-gather-storage-limit_{context}"]
+= Changing the must-gather storage limit
+
+When using the `oc adm must-gather` command to collect data the default maximum storage for the information is 30% of the storage capacity of the container. Once the 30% limit is reached the container is killed and the gathering process stops. Information already gathered will be downloaded to your local storage. You will need either a container with more storage capacity or to adjust the maximum volume percentage before running the must-gather command again.
+
+If you reach the storage limit you will see a message output informing you what happened.
```

Comment:
I'd get rid of the "For example" line below and try to incorporate it into this line, just to make it more concise

```suggestion
If you reach the storage limit, you will see a message output similar to the following example:
```

I added a suggestion from the top of my head, but there are probably better examples elsewhere in the repo